packages/integration-tests/projects/suite-web/tests/wallet/bitcoin-send-form.test.ts

```diff
@@ -51,22 +51,22 @@ describe('Send form for bitcoin', () => {
         cy.getTestElement('@wallet/send/outputs-and-options').matchImageSnapshot('bitcoin-send');
     });
 
-    it('switch display units to satoshis, fill a form in satoshis and send', () => {
```

Comment:
it.skip

Reply:
1d16c6b8c0efb6325aaec426e04dde0f08e6fc35